src/io/EnergyLogWriter.cpp

```diff
@@ -60,6 +60,15 @@ void EnergyLogWriter::endStep(ParticleContainer *particleContainer, DomainDecomp
 	}
 
 	// calculate global values
+#ifdef ENABLE_PERSISTENT
+	auto collComm = make_CollCommObj_AllreduceAdd(domainDecomp->getCommunicator(), nNumMolsGlobalEnergyLocal, UkinLocal, UkinTransLocal, UkinRotLocal);
+	collComm.persistent();
+	unsigned long nNumMolsGlobalEnergyGlobal;
+	double UkinGlobal;
+	double UkinTransGlobal;
+	double UkinRotGlobal;
+	collComm.get(nNumMolsGlobalEnergyGlobal, UkinGlobal, UkinTransGlobal, UkinRotGlobal);
```

Comment:
I think we already talked about this but it would be really cool if we could do something like this:

```suggestion
	auto [nNumMolsGlobalEnergyGlobal, UkinGlobal, UkinTransGlobal, UkinRotGlobal] = collComm.get();
```

Although looking at the implementation behind `get()` I'm not sure how to easily do that 😬 . So if it would be too complicated leave it be.

Reply:
I implemented this functionality in [6f0ab1f](https://github.com/ls1mardyn/ls1-mardyn/pull/329/commits/6f0ab1fd382467b637c07d2a919920ef7ace3736). Thanks again for the idea and all your comments :+1:.